api/models/ccache.go

```diff
@@ -0,0 +1,27 @@
+package models
+
+import (
+	opentracing "github.com/opentracing/opentracing-go"
+)
+
+type CCacheDelete struct {
+	Patterns  []string `json:"patterns" form:"patterns" binding:"Required"`
+	OrgId     int      `json:"orgId" form:"orgId" binding:"Required"`
+	Propagate bool     `json:"propagate" form:"propagate"`
+}
+
+func (cd CCacheDelete) Trace(span opentracing.Span) {
+	span.SetTag("patterns", cd.Patterns)
+	span.SetTag("org", cd.OrgId)
+	span.SetTag("propagate", cd.Propagate)
+}
+
+func (cd CCacheDelete) TraceDebug(span opentracing.Span) {
+}
+
+type CCacheDeleteResp struct {
```

Comment:
the number of errors is interesting, but much more useful I think would be the first error encountered (for each node)